### PR TITLE
docs(v2): add ocpeasy to users

### PR DESCRIPTION
**Pull request by dbrrt**
## Motivation

Adding ocpeasy project to the showcase/users section.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

## Test Plan

Starting the website should display OCPeasy in the showcase part, including `ocpeasy.png` preview.

## Related PRs

N/A

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -420,6 +420,14 @@ const users = [
     source: 'https://github.com/miroiu/nodify',
     tags: ['design'],
   },
+  {
+    title: 'OCPeasy',
+    description: 'Open-source software provisioning, configuration management, and application-deployment tool enabling infrastructure as code on OpenShift.',
+    preview: require('./showcase/ocpeasy.png'),
+    website: 'https://www.ocpeasy.org',
+    source: 'https://github.com/ocpeasy/website',
+    tags: ['design'],
```
Comment (slorber) [inline website/src/data/users.js:429]:
```suggestion
    tags: [],
```

removing this as it's reserved for sites having a custom design

**Review by slorber — APPROVED** [review]
thanks :)

File: website/src/data/users.js (hunk 1 above)
Comment (slorber) [inline website/src/data/users.js:429]:
```suggestion
    tags: [],
```

Comment (slorber) [reply website/src/data/users.js:429]: I'm not able to commit to your PR so please accept my suggestion so that I can merge

Comment (slorber) [reply website/src/data/users.js:429]: going to merge and fix this in another pr

Comment (dbrrt) [reply website/src/data/users.js:429]: @slorber Just saw your message, thanks for that!

**Pull request merged**